beast/physicsmodel/dust/extinction.py

```diff
@@ -503,7 +503,7 @@ def __init__(self):
             np.min([self.ALaw.x_range[1], self.BLaw.x_range[1]]),
         ]
 
-    def function(self, lamb, Av=1, Rv=3.1, Alambda=True, f_A=0.5, **kwargs):
+    def function(self, lamb, Av=1, Rv=3.1, Alambda=True, f_A=0.5, draine_extend=True, **kwargs):
```

Comment:
Why did you decide that the Draine extension should be done by default?  I would have expected we would not do this by default to allow for the measured extinction curve to be used unless one would like to do something like get the ionized photons.

Reply:
Because no one actually goes below F225W and I am only one currently doing. Setting 'draine extension=True' doesn't affect anything above 1695 AA by using the measured extinction curves down to 1695 AA. If I remember correctly, the measured and Draine extinction curves are not that different down to ~1200 AA. If not, please enlighten me. Any cases, I am fine with adding a new parameter in the beast_settings.txt to make this optional.

Reply:
I think we should make this optional.  It is somewhat common to ask for predictions in the GALEX FUV band.  So this would affect that case.